packages/darklang/languageTools/writtenTypes.dark

```diff
@@ -34,7 +34,7 @@ module Darklang =
 
       // Types
       module TypeReference =
-        type BuiltIn =
+        type Builtin =
```

Comment:
Todo: check if this was meant to be renamed from BuiltIn to Builtin

Reply:
Yeah - at least, at this point I believe the majority of the usages are typed this way, so let's be consistent like this.